app/instance-initializers/bugsnag.js

```diff
@@ -1,7 +1,7 @@
 import Ember  from 'ember';
 import config from '../config/environment';
 import { getContext, generateError } from 'ember-cli-bugsnag/utils/errors';
-import { getMetaData } from '../utils/bugsnag';
+import { getMetaData, getUser } from '../utils/bugsnag';
```

Comment:
@binhums Wasn't 100% how to structure things here.

If I add a `getUser` method to `app/utils/bugsnag.js`, it works fine. There's a catch - if a user overrides `utils/bugsnag.js`, they _must_ declare both `getMetaData` and `getUser`. If they omit either one, this import will having nothing reference.

Two options off the top of my head:

1. just go with it, and tell users they have to override both methods in the docs
2. split `getMetaData` and `getUser` into `utils/bugsnag-get-metadata.js` and utils/bugsnag-get-user.js` respectively

Thoughts?